torch/distributed/_tensor/ops/experimental_ops.py

```diff
@@ -41,9 +52,48 @@ def slice_backward_rules(op_schema: OpSchema) -> OutputSharding:
     return OutputSharding(grad_input_spec)
 
 
-@register_prop_rule(aten.bernoulli.default)
-@register_prop_rule(aten.bernoulli_.float)
-def bernoulli_rules(op_schema: OpSchema) -> OutputSharding:
-    input_spec = op_schema.args_schema[0]
-    assert isinstance(input_spec, DTensorSpec)
-    return OutputSharding(input_spec)
+@register_op_strategy(
+    [aten.bernoulli.default, aten.bernoulli.out, aten.bernoulli_.float],
+    schema_info=RuntimeSchemaInfo(static_kwargkey=["out"]),
+)
+def bernoulli_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> OpStrategy:
```

Comment:
iiuc `bernoulli` is just a pointwise operator? I think we can simply adding it to the pointwise list and verify it with tests?